web/packages/demo/www/index.html

```diff
@@ -6,7 +6,6 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>Ruffle Web Demo</title>
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato" />
-        <link rel="stylesheet" href="./index.css" />
```

Comment:
Yup, this is correct, the CSS is inserted directly into the webpage by the build tool, can confirm locally and on https://ruffle.rs/demo/